cli/commands/site/list.ts

```diff
@@ -126,6 +126,8 @@ export async function runCommand( format: 'table' | 'json', watch: boolean ): Pr
 				},
 				{ debounceMs: 500 }
 			);
+
+			process.on( 'SIGINT', disconnect );
```

Comment:
The `studio site list --watch` command would previously not interrupt in response to <kbd>Ctrl+C</kbd>